### PR TITLE
Point fft JIT tests to Halide binary

### PR DESCRIPTION
**Pull request by alexreinking**
The FFT JIT tests didn't have Halide.dll in their path. This fixes that.

### CODE REVIEW

**Review by steven-johnson — APPROVED** [review]

**Comment by steven-johnson** [comment]
...did we decide this didn't need fixing? did it get fixed some other way?

**Comment by alexreinking** [comment]
No. I messed up and thought this was the original fft PR when I was cleaning up my branches earlier today. Restored and reopened...

**Comment by alexreinking** [comment]
Failures are unrelated... low GPU memory for the lens blur filter.

File: src/Generator.cpp [hunk 1]
```diff
@@ -757,7 +757,7 @@ std::string halide_type_to_c_type(const Type &t) {
 
 int generate_filter_main_inner(int argc, char **argv, std::ostream &cerr) {
     const char kUsage[] =
-        "gengen\n"
+        "gengen \n"
```
[the remark this answers is not in the document]
Comment (steven-johnson) [reply src/Generator.cpp:760]: I just touched a string to tickle the buildbots.

**Pull request merged**